Jint/Parser/JavascriptParser.cs

```diff
@@ -62,11 +63,17 @@ public class JavaScriptParser
 
 
         public JavaScriptParser()
+            : this(null)
+        {
+        }
+
+        public JavaScriptParser(Engine engine)
```

Comment:
This is a big issue for me, I'd really like to keep the parser independent from the engine.